tests/unit/libbpf_test.cpp

```diff
@@ -423,6 +424,97 @@ TEST_CASE("disallow prog_array mixed program type values", "[libbpf]")
     REQUIRE(error == -EBPF_INVALID_ARGUMENT);
     REQUIRE(errno == -error);
 
+    // TODO(issue #287): _close(map_fd);
```

Comment:
In the meantime, we can use ebpf_close_fd() to close the map fd.

Reply:
Done